README.md

```diff
@@ -1,33 +1,37 @@
-This package contains dart.js, and previously contained interop.js
+**This package is no longer maintained.**
```

Comment:
looking at the markdown rendering: https://github.com/dart-lang/browser/blob/6823f104e937b5a69b48915c16f5348cfa4f4d1d/README.md

... the emphasis on the deprecation message is less than the dart.js/interop.js headers. It's probably fine, since this is up top, but I wanted to mention. Not sure if there's an easy way to fix that.